test/test-helpers/did-render.ts

```diff
@@ -0,0 +1,13 @@
+import Application from '../../src/application';
+
+async function didRender(app: Application): Promise<void> {
+  return new Promise<void>(resolve => {
+    let watcher = setInterval(function() {
+      if (app['_scheduled']) return;
```

Comment:
Not sure under what conditions this is considered okay. I figured this was better than making `scheduled` public. Happy to change if anyone disagrees.